pkg/api/chunk.go

```diff
@@ -161,8 +163,14 @@ func (s *Service) chunkUploadHandler(w http.ResponseWriter, r *http.Request) {
 
 	if err = wait(); err != nil {
 		s.logger.Debug("chunk upload: sync chunk failed", "error", err)
-		s.logger.Error(nil, "chunk upload: sync chunk failed")
-		jsonhttp.InternalServerError(w, "sync failed")
+		switch {
+		case errors.Is(err, errUnsupportedDevNodeOperation):
+			s.logger.Error(nil, "chunk upload: direct upload not supported in dev mode")
```

Comment:
Would be better to send the err here and in the default case to the log.Error. So we can see it in logs also.